### DISCUSSION TITLE
Please tell me about the rhythm games you've played or heard.

### DISCUSSION THREAD

**Discussion by siyuan-l-huro** [NAIST-SE/AttractingContributors#905] [2023 Attraction Contributions]
Do you know rhythm games?
It's a kind of games that requires you to react with the rhythm of music.
There are thousands titles of rhythm games, available on arcades, game consoles and PCs.
What kind of rhythm games have you played or heard of? 
Please tell me by pulling requests at https://github.com/siyuan-l-huro/AttractingContributors-RhythmGames

**Comment by bconfessor** [comment]
I like Guitar Hero, played it a lot on the DS.
I sent you a pull request! 
Could you please tell me your favorite song?? 
https://github.com/bconfessor/FavoriteSong/blob/main/README.md
